File: org.opencompare/api-java/src/main/java/org/opencompare/api/java/io/ImportCell.java
package org.opencompare.api.java.io;

import org.opencompare.api.java.Value;

/**
 * Created by gbecan on 08/12/15.
 */
public class ImportCell extends IOCell {

    private Value interpretation;

    public ImportCell() {
        super("");
    }

    public ImportCell(String content) {
        super(content);
    }

    public ImportCell(String content, String rawContent) {
        super(content, rawContent);
    }

    public ImportCell(String content, int rowspan, int colspan) {
        super(content, rowspan, colspan);
    }

    public ImportCell(String content, String rawContent, int rowspan, int colspan) {
        super(content, rawContent, rowspan, colspan);
    }

    public Value getInterpretation() {
        return interpretation;
    }

    public void setInterpretation(Value interpretation) {
        this.interpretation = interpretation;
    }
}

File: org.opencompare/api-scala/src/main/scala/org/opencompare/api/scala/io/ImportCell.scala
package org.opencompare.api.scala.io

import org.opencompare.api.scala.Value

class ImportCell(
                  initContent : String = "",
                  initRawContent : String = "",
                  initRowspan : Int = 1,
                  initColspan : Int = 1)
  extends IOCell(initContent, initRawContent, initRowspan, initColspan) {

  var interpretation : Option[Value] = None


}
